src/drive/locales/en.json

```diff
@@ -18,7 +18,7 @@
       "https://play.google.com/store/apps/details?id=io.cozy.drive.mobile",
     "link-client-ios":
       "https://itunes.apple.com/us/app/cozy-drive/id1224102389?mt=8",
-    "link-client-web": "https://cozy.io/fr/try-it?from=albumsharing"
+    "link-client-web": "https://cozy.io/fr/try-it"
```

Comment:
Not really related with this PR, sorry, but I was too lazy to do a separate PR just for this.

Reply:
Shouldn't that be `"link-client-web": "https://cozy.io/en/try-it"`

Reply:
Good catch, it shouldn't have a language at all actually.